docs/sources/fundamentals/architecture/components/_index.md

```diff
@@ -233,3 +233,5 @@ factor, it is possible that the querier may receive duplicate data. To resolve
 this, the querier internally **deduplicates** data that has the same nanosecond
 timestamp, label set, and log message.
 
+At read path, replication factor also plays a role here. For example with `replication_factor` of `3`, we require that two queries to be running. 
```

Comment:
@JStickler , I'd love to have the "replication factor" on this sentence to be linked to the replication factor heading.
Though, I'm not sure how to do that.

This is the heading I'm referring to: https://grafana.com/docs/loki/latest/fundamentals/architecture/components/#replication-factor

Reply:
```suggestion
At read path, replication factor also plays a role here. For example with `{{< relref "#replication-factor" >}}` of `3`, we require that two queries to be running. 
```